GNUmakefile

```diff
@@ -245,6 +245,7 @@ test-nomad: dev ## Run Nomad test suites
 	@echo "==> Running Nomad test suites:"
 	@NOMAD_TEST_RKT=1 \
 		go test $(if $(VERBOSE),-v) \
+			-v \
```

Comment:
remove this, because all the changes I made in PR #3626 were because we thought that Travis should not have verbose logging.

Reply:
Ack, thanks for catching that. The entire 7b8d9d4 commit was intended to be a temporary test and reverted. Killing it now.